tools/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ add_subdirectory(compile_tool)
 
 if(ENABLE_SAMPLES)
     add_subdirectory(legacy/benchmark_app)
+    add_subdirectory(legacy/c)
```

Comment:
I don't think that we need to have legacy c samples. Let's remove it

Reply:
ok, I will remove legacy c sample completely.

Reply:
done